src/website/src/pages/ec/components/news-ticker/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Messages
```

Comment:
i was wondering the reason why i was not finding the news ticker in the menu on the left.. :)